packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -109,8 +109,8 @@ const disabilityAssessments =  [
 	{
 		identifier: "insane",
 		nonInclusivePhrases: [ "insane" ],
-		inclusiveAlternatives: "<i>wild, confusing, unpredictable, impulsive, reckless, out of control, " +
-			"unbelievable, incomprehensible, irrational, nonsensical, outrageous, ridiculous</i>",
+		inclusiveAlternatives: "<i>wild, confusing, unpredictable, impulsive, reckless, out of control " +
```

Comment:
Doesn't this line need an extra "," at the end, like before?

Reply:
yes it does! Thank you, will fix it